net/fail2ban/Makefile

```
@@ -0,0 +1,115 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fail2ban
+PKG_VERSION:=0.11.2
+PKG_RELEASE:=2
```

```suggestion
PKG_RELEASE:=1
```

See: https://github.com/seb-m/pyinotify/blob/master/COPYING